src/components/CreatePoll/CreatePollTime.js

```diff
@@ -47,28 +50,40 @@ export default function CreatePollTime({
           onChange={t =>
             handleParentState({
               start: t,
```

Comment:
it seems like we're passing the raw `t` object for `start` and `t.getTime()` for `end` -- is this ok? @sirromdev

Reply:
fixed